governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx

```diff
@@ -356,6 +379,275 @@ const Proposal = ({
                 </div>
               </>
             ) : null}
+            {instruction instanceof WormholeMultisigInstruction ? (
+              <div className="col-span-4 my-2 space-y-4 bg-darkGray2 p-4 lg:col-span-3">
+                <h4 className="h4">Wormhole Instructions</h4>
```

Comment:
Can you display program name here for the parsed instruction `"Program : Pyth oracle"`